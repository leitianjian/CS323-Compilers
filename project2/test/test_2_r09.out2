Error type 9 at Line 12: the functionâ€™s arguments mismatch the declared parameters
Error type 8 at Line 12: function return type is not matched
