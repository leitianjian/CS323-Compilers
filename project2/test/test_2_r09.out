Error type 9 at line 12: the functionâ€™s arguments mismatch the declared parameters
